src/utils/constants.ts

```diff
@@ -255,4 +269,29 @@ export default {
   TEMPLATE_BUTTON_SAVE: 'Save',
   TEMPLATE_BUTTON_ADD: 'Add',
   TEMPLATE_APPEARS_IN: 'Appears in',
+  OUTCOMES_TABLE_PAGINATION_TEXT: 'Items per page',
+  APP_LINK_LINKED_APPLICATIONS: 'Linked Applications',
+  APP_LINK_NAME: 'Name',
+  APP_LINK_SERIAL: 'Serial',
+  APP_LINK_TYPE: 'Application Type',
+  APP_LINK_COMPLETED: 'Date Completed',
+  LOOKUP_ERROR_NOT_FOUND: 'No lookup-table found!',
+  LOOKUP_ERROR_TITLE: 'Error loading lookup-table',
+  LOOKUP_TABLE_HEADER_LABEL: 'Label',
+  LOOKUP_TABLE_HEADER_NAME: 'Table Name',
+  LOOKUP_TABLE_HEADER_ACTIONS: 'Actions',
+  LOOKUP_TABLE_VIEW: 'View lookup table',
+  LOOKUP_TABLE_IMPORT: 'Import lookup table',
+  LOOKUP_TABLE_IMPORT_INTO: 'Import into Lookup-table: %1',
+  LOOKUP_TABLE_IMPORT_OTHER: 'Import another CSV',
+  LOOKUP_TABLE_DOWNLOAD: "Download '%1' table",
+  LOOKUP_TABLE_FIELD_NAME: 'Database Field-name',
+  LOOKUP_TABLE_LABEL: 'Lookup-table Label',
+  LOOKUP_TABLE_NAME: 'Table name',
+  LOOKUP_TABLE_ADD: 'Add new Lookup-table',
+  LOOKUP_TABLE_TITLE: 'Lookup Table: %1',
+  LOOKUP_TABLE_CONTENTS: 'View individual lookup-table and its contents',
+  LOOKUP_TABLE_SUBTITLE: 'This page contains the list of all lookup-tables',
+  LOOKUP_TABLE_SUCCESS: "Lookup table '%1' has been successfully %2.",
```

Comment:
I don't think we should use substitutions that replace with other language strings. Isn't that making an assumption about the structure of the language?

I think this would be safer as two complete sentences:
`"Lookup table '%1' has been successfully loaded`
`"Lookup table '%1' has been successfully created`

Because different languages might have different ways to order those sentences, they might not just drop in single words in the same way. Also (and perhaps more importantly) it's easier for the person translating. If they see a string that just has "loaded" or "created" out of context, they might put the wrong word in without knowing the sentence context.

Like, if I'm translating the word "loaded" out of context, I don't know if it's referring to "loaded" as in software, "loaded" as in "stacked up", or "loaded" as in "drunk". If I was translating the whole sentence, it would be clear.

So, as a general rule, I think localised strings should be full phrases/sentences as much as possible, even if there is some repetition in most of the sentence. The translator shouldn't have to cross-check against other strings and perform mental gymnastics to figure out what's being said from multiple strings. Do you agree?